backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5684510_sys_gh15086_JobRouter_Instance_ID.sql

```diff
@@ -0,0 +1,115 @@
+;
+
+-- 2023-04-06T20:50:32.087Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(543939) 
+;
+
+-- 2023-04-06T20:50:32.213Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=713823
+;
+
+-- 2023-04-06T20:50:32.329Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(713823)
+;
+
+-- 2023-04-06T20:51:11.385Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(582231) 
+;
+
+-- 2023-04-06T20:51:11.494Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=713823
+;
+
+-- 2023-04-06T20:51:11.586Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(713823)
+;
+
+-- UI Element: Invoice (Vendor)_OLD(183,D) -> Invoice(290,D) -> main -> 20 -> Externe ID
+-- Column: C_Invoice.ExternalId
+-- 2023-04-06T20:53:16.849Z
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_ElementGroup_ID,AD_UI_Element_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayedGrid,IsDisplayed_SideList,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNoGrid,SeqNo_SideList,Updated,UpdatedBy,WidgetSize) VALUES (0,713823,0,290,540226,616643,'F',TO_TIMESTAMP('2023-04-06 21:53:15','YYYY-MM-DD HH24:MI:SS'),100,'Y','N','N','Y','N','N','N',0,'Externe ID',50,0,0,TO_TIMESTAMP('2023-04-06 21:53:15','YYYY-MM-DD HH24:MI:SS'),100,'S')
+;
+
+-- Field: Invoice (Vendor)_OLD(183,D) -> Invoice(290,D) -> Externe ID
+-- Column: C_Invoice.ExternalId
+-- 2023-04-06T21:01:04.970Z
+UPDATE AD_Field SET DisplayLogic='@ExternalId@!NULL',Updated=TO_TIMESTAMP('2023-04-06 22:01:04','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=713823
+;
+
+-- Field: Invoice (Vendor)_OLD(183,D) -> Invoice(290,D) -> Externe ID
+-- Column: C_Invoice.ExternalId
+-- 2023-04-06T21:03:11.364Z
+UPDATE AD_Field SET DisplayLogic='', ReadOnlyLogic='@ExternalId@!NULL',Updated=TO_TIMESTAMP('2023-04-06 22:03:11','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=713823
+;
+
+-- Column: C_Invoice.ExternalId
+-- 2023-04-06T21:04:08.565Z
+UPDATE AD_Column SET IsAlwaysUpdateable='Y',Updated=TO_TIMESTAMP('2023-04-06 22:04:08','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=572375
+;
+
+-- Field: Invoice (Vendor)_OLD(183,D) -> Invoice(290,D) -> Externe ID
+-- Column: C_Invoice.ExternalId
+-- 2023-04-06T21:09:05.735Z
+UPDATE AD_Field SET ReadOnlyLogic='',Updated=TO_TIMESTAMP('2023-04-06 22:09:05','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=713823
+;
+
+-- Field: Invoice (Vendor)_OLD(183,D) -> Invoice(290,D) -> Externe ID
+-- Column: C_Invoice.ExternalId
+-- 2023-04-06T21:13:51.918Z
+UPDATE AD_Field SET IsAlwaysUpdateable='', ReadOnlyLogic='@ExternalId@!NULL',Updated=TO_TIMESTAMP('2023-04-06 22:13:51','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=713823
```

Comment:
We do not want this logic in the public repo. The field shall be editable only when the record is not yet processed.